src/tools/kata-ctl/src/log_parser/README.md

```diff
@@ -43,6 +43,7 @@ The most valuable command line options are listed below:
 - `--output-format <OUTPUT_FORMAT>` Sets the format of the output. Defaults to `json`, and can be set to `csv`, `json`, `ron`, `text`, `toml`, `xml`, and `yaml`.
 - `-q, --quiet` Will not print invalid log entry errors to stderr.
 - `-s, --strict` Any invalid log entry will halt the program.
+- `-h, --help`, Displays a listing of all cli options.
```

Comment:
@gabevenberg could you update this to CLI? I think the lowercase is causing it to fail, I just checked and CLI but not cli is in the spell check dictionary and acronym list